core/primitives/res/epoch_configs/mainnet/81.json

```diff
@@ -141,5 +141,11 @@
   "max_inflation_rate": [
     1,
     40
-  ]
+  ],
+  "protocol_reward_rate": [
+    1,
+    10
+  ],
+  "protocol_treasury_account": "near",
+  "num_blocks_per_year": 31536000
```

Comment:
I don't think the `num_blocks_per_year` is used anywhere in the codebase. We can simply remove it. (Ofc, we unfortunately won't be able to change genesis config)